molgenis-web/src/test/java/org/molgenis/web/PluginInterceptorTest.java

```diff
@@ -21,13 +23,21 @@
 	private Ui molgenisUi;
 	private PermissionService permissionService;
 	private Authentication authentication;
+	private DataService dataService;
 
 	@BeforeMethod
-	public void setUp()
+	public void beforeMethod()
```

Comment:
In MOLGENIS this method is usually name setUpBeforeMethod, in suggest renaming it.

Reply:
Actually:

setUpBeforeMethod: 88 occurrences in 88 files
beforeMethod: 100+ occurrences in 100+ files

But I agree we should be consistent in naming these @BeforeMethod and @BeforeClass annotated classes